src/renderer/models/types/networkMetrics.ts

```diff
@@ -0,0 +1,20 @@
+import {ContainerType, ListType, NumberUintType} from "@chainsafe/ssz";
+import {StringType} from "./basic";
+import {NetworkMetric, NetworkMetrics} from "../networkMetrics";
+
+const MILLION = 1000000;
+
+export const NetworkMetricType = new ContainerType<NetworkMetric>({
+    fields: {
+        url: new StringType(),
+        code: new NumberUintType({byteLength: 32}),
+        latency: new NumberUintType({byteLength: 32}),
+        time: new NumberUintType({byteLength: 32}),
+    },
+});
+
+export const NetworkMetricsType = new ContainerType<NetworkMetrics>({
+    fields: {
+        records: new ListType({elementType: NetworkMetricType, limit: MILLION}),
```

Comment:
I meant you could do something like this: https://github.com/ChainSafe/ssz/blob/master/src/types/composite/array.ts

```
export class UnboundedArray extends CompositeType {
   constructor(opts: {type}) {
   }

   serialize() {
     //impl
    }

       deserialize() {
     //impl
    }

   toJson() {
     //impl
    }

      fromJson() {
     //impl
    }

}
```

and use it instead of ListType